src/score_hv/stats_utils.py

```diff
@@ -7,43 +8,106 @@
     gridcell_area_data - xarray variable containing gridcell areas, in the
     same shape as xarray_variable
 """
-
+import sys,os
 import numpy as np
+import pytest
+import pdb
+from netCDF4 import Dataset
+from pathlib import Path
 
-def area_weighted_mean(xarray_variable, gridcell_area_weights):  
-    """returns the gridcell area weighted mean of xarray_variable and checks
-    that gridcell_area_weights are valid
+class var_stats:
```

Comment:
I think the class needs to be named "var_statsCatalog," which is how it is referenced elsewhere

Reply:
Agreeing that it needs to match. I would suggest VarStatsCatalog as a similar naming structure to other classes. Either way, this needs to match what gets imported in daily_bfg.py